docs/index.asciidoc

```diff
@@ -339,6 +340,18 @@ input {
 }
 --------------------------------------------------
 
+[id="plugins-{type}s-{plugin}-event_loop_threads"]
+===== `event_loop_threads`
+
+* Value type is <<number,number>>
+* Defaults to 0.
```

Comment:
When setting `0`, the actual default is `available_processors * 2`

Reply:
Updated.